ext/pdo_mysql/tests/inc/mysql_pdo_test.inc

```diff
@@ -1,6 +1,6 @@
 <?php
-require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'config.inc');
-require_once(dirname(__FILE__) . '/../../../ext/pdo/tests/pdo_test.inc');
+require_once(dirname(__FILE__) . DIRECTORY_SEPARATOR . 'init.inc');
+require_once(dirname(__FILE__) . str_replace('/', DIRECTORY_SEPARATOR, '/../../../../ext/pdo/tests/pdo_test.inc'));
 
```

Comment:
```suggestion
require_once __DIR__ . DIRECTORY_SEPARATOR . 'init.inc';
require_once dirname(__DIR__, 4) . '/ext/pdo/tests/pdo_test.inc';
```
No need to convert paths for Windows, also do not use parenthesis around require as it is not a function but a language construct and the semantics are slightly different.